MobileCenter/MobileCenter/Internals/Model/MSAbstractLog.m

```diff
@@ -28,8 +27,8 @@ - (NSMutableDictionary *)serializeToDictionary {
   }
   if (self.toffset) {
 
-    // Set the toffset relative to current time. The toffset needs to be up to date.
-    NSInteger now = [[NSDate date] timeIntervalSince1970];
+    // Set the toffset relative to current time. The toffset needs to be up to date.    
+    NSInteger now = ([[NSDate date] timeIntervalSince1970] * 1000);
```

Comment:
NSInteger Max is big enough as it's actually defined as:
`#define NSIntegerMax    LONG_MAX`